archive_submission_guide.md

```diff
@@ -6,6 +6,17 @@ We assume you have some basic knowledge about using a command line software like
 
 !> This documentation only covers submissions to the [Poseidon Community Archive](archive_overview). Look [here](minotaur) for the submission process to the [Poseidon Minotaur Archive](archive_overview).
 
+## Archive curation roles
+
+To manage package submissions and modifications in our archives, we define the following roles, which are synonymous to the respective roles within github:
+
+1. Assignees: A package is submitted by a single author, with a github account. This user is tagged as "Assignee" in the github interface. The same holds for the modification of an existing package: Here, the "assignee" is the user who authors a Pull Request to change a given package. Assignees are specific per package. An assignee is responsible for bringing the package into shape, and responding to review requests.
+
+2. Reviewers: A Pull request for a new or modified package is reviewed by one or more users. Reviewers will often be recruited from the Poseidon Core Team, but can also encompass other relevant users, for example if they have special knowledge on the package, or otherwise expertise. Guidelines for reviewers and assignees overlap, and are summarised below. Reviewers are asked to ensure that all checklist items are covered.
+
+3. Editors: Editors are not assigned per package, but per repository. Currently, editors for the [Community Archive](#the-poseidon-community-archive-pca) are users [@Aygal](https://github.com/AyGhal) and [@nevrome](https://github.com/nevrome). Editor for the [Minotaur Archive](#the-poseidon-minotaur-archive-pma) is user [@TCLamnidis](https://github.com/TCLamnidis).
```

Comment:
```suggestion
3. Editors: Editors are not assigned per package, but per repository. **Editors are responsible for merging Pull Requests into their respective archives, and maintaining those archives.** Currently, editors for the [Community Archive](#the-poseidon-community-archive-pca) are users [@Aygal](https://github.com/AyGhal) and [@nevrome](https://github.com/nevrome). Editor for the [Minotaur Archive](#the-poseidon-minotaur-archive-pma) is user [@TCLamnidis](https://github.com/TCLamnidis).
```
There was no explanation of what the editor's role actually entails, so I added my understanding of it.

Reply:
Yes, good thing! I will also add that editors recruit reviewers for a submission.